www/js/control/ProfileSettings.jsx

```diff
@@ -25,6 +25,7 @@ import { AppContext } from "../App";
 import { shareQR } from "../components/QrCode";
 import { storageClear } from "../plugin/storage";
 import { getAppVersion } from "../plugin/clientStats";
+import { useSchedulerHelper } from "../splash/notifScheduler";
```

Comment:
If you export `update`, `getScheduledNotifs`, `getReminderPrefs`, and `setReminderPrefs` directly from `notifScheduler.ts`, you can import them as individual functions instead of `useSchedulerHelper`